x-pack/plugins/enterprise_search/public/applications/shared/kibana_chrome/generate_breadcrumbs.test.ts

```diff
@@ -40,6 +41,9 @@ describe('useGenerateBreadcrumbs', () => {
       { text: 'Groups', path: '/groups' },
       { text: 'Example Group Name', path: '/groups/{id}' },
       { text: 'Source Prioritization', path: '/groups/{id}/source_prioritization' },
+      // Note: We're still generating a path for the last breadcrumb even though useEuiBreadcrumbs
+      // will not render a link for it. This is because it's easier to keep our last-breadcrumb-specific
+      // logic in one place, & this way we still have a current path if (for some reason) we need it later.
```

Comment:
FYI, just wanted to highlight this. There were a few places I could have added a last-breadcrumb logic check - in either `useEuiBreadcrumbs` or `useGenerateBreadcrumbs`/`useEnterpriseSearchBreadcrumbs`, etc.. I opted for only adding "last breadcrumb" logic in `useEuiBreadcrumbs` because doing so required the fewest amount of changes/lines touched.

Reply:
Yeah, good call, I started doing it in `useGenerateBreadcrumbs`, and went down the rabbit hole :D

Reply:
HAHA yeah if it makes you feel better I did too and then I was like "wait a minute..." 🤦‍♀️